cookbook/logging/monolog_email.rst

```diff
@@ -121,20 +129,23 @@ it is only triggered when the action level, in this case ``critical`` is reached
 The ``critical`` level is only triggered for 5xx HTTP code errors. If this level
 is reached once, the ``fingers_crossed`` handler will log all messages
 regardless of their level. The ``handler`` setting means that the output
-is then passed onto the ``buffered`` handler.
+is then passed onto the ``deduplicated`` handler.
 
 .. tip::
 
     If you want both 400 level and 500 level errors to trigger an email,
     set the ``action_level`` to ``error`` instead of ``critical``. See the
     code above for an example.
 
-The ``buffered`` handler simply keeps all the messages for a request and
-then passes them onto the nested handler in one go. If you do not use this
-handler then each message will be emailed separately. This is then passed
-to the ``swift`` handler. This is the handler that actually deals with
-emailing you the error. The settings for this are straightforward, the
-to and from addresses and the subject.
+The ``deduplicated`` handler simply keeps all the messages for a request and
+then passes them onto the nested handler in one go, but only if the records are
+unique over a given period of time (60 seconds by default). If the records are
+duplicates they are simply discarded. Adding this handler reduces the amount of
+notifications to a manageable level, specially in critical failure scenarios.
+The messages are then passed to the ``swift`` handler. This is the handler that
```

Comment:
I would add a blank line before this sentence to have it rendered as its own paragraph to improve readability.